frontend/src/scenes/feature-flags/FeatureFlag.tsx

```diff
@@ -692,12 +693,6 @@ function FeatureFlagRollout({ readOnly }: FeatureFlagReadOnlyProps): JSX.Element
                                                             }
                                                         }
                                                     }}
-                                                    style={{
```

Comment:
Anyway to control border with lemon input?

Reply:
Not yet, but PR incoming with `status="danger"` support! Feel free to post any Lemon UI questions in #dev, as I'll be picking up these types of system-wide things

Reply:
Here: https://github.com/PostHog/posthog/pull/13686